cmd/test.go

```diff
@@ -73,12 +77,24 @@ const (
 	categoryScoreC categoryScore = "C"
 )
 
+type testResultError struct{ error }
```

Comment:
Even if it's unexported, can we have some godoc here explaining the purpose of this struct?